phpstan.neon

```diff
@@ -1,6 +1,9 @@
+includes:
+    - phpstan-baseline.neon
+
 parameters:
-  level: 5
-  ignoreErrors:
-    -
-      message: '!Parameter #3 \$namespace of method XMLWriter::startElementNs\(\) expects string, null given.!'
-      path: lib/Writer.php
+  level: 6
+  phpVersion: 70234 # PHP 7.2.34
```

Comment:
using a fixed php version to make it possible to create a baseline locally on different php versions.

doesn't matter much which version we use here. I am using 7.2 as its on the lower end but 7.1 seemed too old for my feelings